workflow-bridge-lambda/src/index.ts

```diff
@@ -37,13 +39,15 @@ async function getPinboardByComposerId(composerId: string) {
   return { ...data, status: data.externalData?.status };
 }
 
-async function getAllPinboards() {
+async function getAllPinboards(searchText?: string) {
   const fields = ["id", "title", "composerId"].join(",");
 
+  const searchQueryParamClause = searchText ? `&text=${searchText}` : "";
```

Comment:
does `searchText` need URL encoding?

Reply:
although it didn't cause any problems so far, still a good idea - done